README.md

```diff
@@ -38,6 +38,40 @@ Bullet• is triggered by Dagger‡'s `@Component` and `@Subcomponent` annotatio
 so you only need to put Bullet• in your processor path to get it to work;
 no need to change anything to your component interfaces.
 
+The bullet-${bullet.version}.jar needs to be included in the application's runtime. In order to activate code generation the bullet-compiler-${dagger.version}.jar needs to be included in your build at compile time.
+
+In a Maven project, you might include the runtime in the dependencies section of your pom.xml (replacing ${dagger.version} with the appropriate current release), and the dagger-compiler artifact as an "optional" or "provided" dependency:
+
+```xml
+<dependencies>
+ <dependency>
+  <groupId>net.ltgt.dagger</groupId>
+  <artifactId>bullet</artifactId>
+  <version>${bullet.version}</version>
+ </dependency>
+ <dependency>
+  <groupId>net.ltgt.dagger</groupId>
+  <artifactId>bullet-compiler</artifactId>
+  <version>${bullet.version}</version>
+  <optional>true</optional>
+ </dependency>
+</dependencies>
+```
+
+In a Gradle project is similar:
+
+```groovy
+compile 'net.ltgt.dagger:bullet:0.20'
+provided 'net.ltgt.dagger:bullet-compiler:0.20'
```

Comment:
`provided` is only provided by the `war` plugin (and some other plugins following the same convention). I'd (unsurprisingly) suggest using the [`net.ltgt.apt`](https://plugins.gradle.org/plugin/net.ltgt.apt) plugin instead.